### DISCUSSION TITLE
Can the .ipa released by flash be submitted to the app store? The games I submitted in the past two years are always rejected and the account is blocked

### DISCUSSION THREAD

**Discussion by qywlgame** [airsdk/Adobe-Runtime-Support#1956] [General]
In 2019 and 2020, I submitted some flash-developed games to the appstore, but none of the games were released successfully, they were always rejected, and the account was banned after 14 days. Can anyone submit successfully for release now?thanks!

**Comment by esdebon** [comment]
In 2020 I uploaded applications made in flash (Animate) to the store without problems, can you tell us the details of the rejection?

**Reply by marchbold** [reply]
Yeah we have been submitting apps for years without any issues. If you provide more information on what went wrong I'm sure we can help you through it.

**Reply by qywlgame** [reply]
issues:"Hello,Upon further review of the activity associated with your Apple Developer Program membership, we have determined that your membership,ora membership associated with your account, has been usedfordishonestorfraudulent activity. Therefore, your Apple Developer Program account has been flaggedforremoval.Ifyou would like to appeal this termination to the App Review Board, you mustdoso within14calendar days. When submitting your appeal, be sure to select"I would like to appeal an app rejection or app removal"from the drop-down menu on the Contact the App Review Team page. Otherwise, your Apple Developer Program membership will be terminatedandall apps associated with your account will be removed from the App Store.We found that your app is in direct violation of the Apple Developer Program License Agreement, which states:"You will not, directly or indirectly, commit any act intended to interfere with the Apple Software or Services, the intent of this Agreement, or Apple’s business practices including, but not limited to, taking actions that may hinder the performance or intended use of the App Store, Custom App Distribution, or the Program (e.g., submitting fraudulent reviews of Your own Application or any third party application, choosing a name for Your Application that is substantially similar to the name of a third party application in order to create consumer confusion, or squatting on application names to prevent legitimate third party use).”"

**Reply by quanticoder** [reply]
I think the problem was not FLASH, but the name you gave to your app, or maybe the id. What name did you use?

**Reply by marchbold** [reply]
Yeah I would say there was something wrong with the content or naming of your app to get a message like that.  Is something similar to your app already available on the store? They have a pretty strict policy about copying other applications these days.

**Reply by qywlgame** [reply]
Isn't it because the air sdk code is the same? Was judged to be duplicate the app？
Game name:"Baby Learning Numbers" "Prince Find Treasure game" ...
sometimes issues:"We are unable to continue this app’s review because your Apple Developer Program account is currently under investigation for not following the App Store Review Guidelines’ Developer Code of Conduct.
Common practices that may lead to an investigation include, but are not limited to:
Inaccurately describing an app or service
Misleading app content
Engaging in inauthentic ratings and reviews manipulation
Providing misleading customer support responses
Providing misleading responses in Resolution Center
Engaging in misleading purchasing or bait-and-switch schemes
Engaging in other dishonest or fraudulent activity within or outside of the app
During our investigation, we will not review any apps you submit. Once we have completed our investigation, we will notify you via Resolution Center."

**Reply by marchbold** [reply]
No that has nothing to do with the AIR SDK code. 

Something specific about your app or what you are doing on the app store.

**Reply by GeneralVimes** [reply]
I think this is "Baby" part which hurts the most. There are very strict guidelines of what should and what should not be in the games targeted at babies. AIR SDK has nothing to do with it

**Reply by esdebon** [reply]
They say: "we will not review any apps you submit", So they didn't even get to review the app, and it was all about the name or the id.

Use an ID with a reverse domain like com.apple or com.icloud, com.me, com.mac It's identity theft

**Reply by qywlgame** [reply]
Ok!Thanks everyone, I will try it again, if there is any problem, I will ask for your help again.
